worker/uniter/runner/jujuc/juju-log_test.go

```diff
@@ -22,59 +19,6 @@ type JujuLogSuite struct {
 
 var _ = gc.Suite(&JujuLogSuite{})
 
-func assertLogs(c *gc.C, ctx jujuc.Context, writer *loggo.TestWriter, unitname, badge string) {
```

Comment:
This test should be re-implemented. However, the juju/cmd package now replaces default logo writer in its main routine. This global overwrite in the main routine wipes out the writer created by this test causing it to fail. A possible fix is to update juju/loggo to allow users to query the default writer; as far as I know the ability is not exported by the package.